docs/user_guide.rst

```diff
@@ -54,7 +65,8 @@ You can specify the base URL on the command line:
 Using a Configuration File
 --------------------------
 
-You can specify the base URL using a configuration file:
+You can specify the base URL using a
+:ref:`configuration file <configuration-files>`:
```

Comment:
Yeah, this is just an issue where GitHub is rendering the file as HTML rather than building the documentation.